systemtest/src/test/java/io/strimzi/systemtest/security/SecurityST.java

```diff
@@ -675,76 +676,110 @@ void testAutoRenewCaCertsTriggerByExpiredCertificate(ExtensionContext extensionC
 
     @ParallelNamespaceTest
     @Tag(INTERNAL_CLIENTS_USED)
-    void testCertRenewalInMaintenanceWindow(ExtensionContext extensionContext) {
-        final String namespaceName = StUtils.getNamespaceBasedOnRbac(namespace, extensionContext);
-        final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
-        final String topicName = mapWithTestTopics.get(extensionContext.getDisplayName());
-        final String secretName = KafkaResources.clusterCaCertificateSecretName(clusterName);
-        final String userName = mapWithTestUsers.get(extensionContext.getDisplayName());
-        final LabelSelector kafkaSelector = KafkaResource.getLabelSelector(clusterName, KafkaResources.kafkaStatefulSetName(clusterName));
+    void testCertRenewalInMaintenanceTimeWindow(ExtensionContext extensionContext) throws InterruptedException {
+        final TestStorage testStorage = new TestStorage(extensionContext, namespace);
+        final String clusterSecretName = KafkaResources.clusterCaCertificateSecretName(testStorage.getClusterName());
+        final String clientsSecretName = KafkaResources.clientsCaCertificateSecretName(testStorage.getClusterName());
 
         LocalDateTime maintenanceWindowStart = LocalDateTime.now().withSecond(0);
         long maintenanceWindowDuration = 14;
-        maintenanceWindowStart = maintenanceWindowStart.plusMinutes(5);
+        maintenanceWindowStart = maintenanceWindowStart.plusMinutes(5).plusMinutes(Duration.ofMillis(Constants.SAFETY_RECONCILIATION_INTERVAL).toMinutes());
```

Comment:
You don't rely on this anymore in the test as much as you did before. So I think you can just set it to something long time int he future to avoid any timing issues.

Reply:
Forgotten bit, thank you.